### DISCUSSION TITLE
How to increase number of days in fixed time span

### DISCUSSION THREAD

**Discussion by Vaibhav2121** [nasa/openmct#6777] [Community Q&A]
There is limit of one day in fixed time span. How can we increase it in Open MCT.

**Comment by ja51d** [comment]
The time span limits are defined in the configuration of the Time Conductor plugin during installation. If you want to allow for longer durations in the fixed time span, you can add custom presets to the menuOptions.

Try updating your openmct.install call for the Time Conductor like this:

[timc.js](https://github.com/user-attachments/files/27810163/timc.js)


This will add '7 Days' and '30 Days' to your dropdown menu. If you are trying to manually select dates in the calendar that are more than a day apart and it's being blocked, ensure that your timeSystem isn't imposing a maximum duration constraint.
